include/geode/geometry/detail/aabb_impl.h

```diff
@@ -453,70 +453,87 @@ namespace geode
         const auto box_half_extent = ( box.max() - box.min() ) / 2.;
 
         // Transform the ray to the aligned-box coordinate system.
-        const auto ray_origin = ray.origin() - box_center;
+        const auto ray_translated_origin = ray.origin() - box_center;
 
         for( const auto i : geode::Range{ dimension } )
         {
-            if( std::fabs( ray_origin.value( i ) ) > box_half_extent.value( i )
-                && ray_origin.value( i ) * ray.direction().value( i ) >= 0. )
+            if( ( std::fabs( ray_translated_origin.value( i ) )
+                    - box_half_extent.value( i ) )
+                    > global_epsilon
+                && ray_translated_origin.value( i ) * ray.direction().value( i )
+                       >= global_epsilon )
             {
                 return false;
             }
         }
-        return line_box_intersection(
-            ray_origin, ray.direction(), box_half_extent );
+        return line_box_intersection( ray, box );
```

Comment:
in the previous implementation you gave the translated ray as paremeter but not anymore. Why not giving the translated line?